e2e/actions/actions.go

```diff
@@ -65,7 +65,8 @@ func actionRun(t *testing.T) {
 	for _, tt := range tests {
 		e2e.RunSingularity(
 			t,
-			tt.name,
+			testenv.CmdPath,
+			e2e.WithSubtest(tt.name),
```

Comment:
I like that with name argument, just it would be more expressive with `AsSubTest(name)`

Reply:
will do